python/libraries/bimvee/importRpgDvsRos.py

```diff
@@ -89,11 +91,13 @@ def bimveeTypeForRosbagType(rosbagType):
     if rosbagType == 'esim_msgs_OpticFlow': return 'flowMap'
     if rosbagType == 'geometry_msgs_PoseStamped': return 'pose6q'
     if rosbagType == 'geometry_msgs_Transform': return 'pose6q'
+    if rosbagType == 'geometry_msgs_TransformStamped': return 'pose6q'
     if rosbagType == 'geometry_msgs_TwistStamped': return 'twist'
     if rosbagType == 'sensor_msgs_CameraInfo': return 'cam'
     if rosbagType == 'sensor_msgs_Image': return 'frame'
     if rosbagType == 'sensor_msgs_Imu': return 'imu'
     if rosbagType == 'sensor_msgs_PointCloud2': return 'point3'
+    if rosbagType == 'tf_tfMessage': return 'pose6q'
```

Comment:
Don't think it should be decoded as 'pose6q'. There is a layer in between. geometry_msgs/TransformStamped is inside tf/tfMessage. In the case of ESIM, /tf has 2 geometry messages (1 for body and 1 for cam0).

Reply:
Thanks for this observation - I've fixed this in importRosbag repo - now frame_id and child_frame_id are also imported and they can be used to separate out the poses for different bodies in a post-processing step if desired, similarly to what happens in importVicon.